src/org/ods/component/Context.groovy

```diff
@@ -6,7 +6,7 @@ import com.cloudbees.groovy.cps.NonCPS
 import groovy.json.JsonSlurper
 import groovy.json.JsonOutput
 
-@SuppressWarnings('MethodCount')
+@SuppressWarnings(['MethodCount', 'PublicMethodsBeforeNonPublicMethods'])
```

Comment:
If you merge `master` into your branch, this should not be needed anymore. Would be nice to not suppress to avoid wrong order in the future.